flank-scripts/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ shadowJar.apply {
     }
 }
 // <breaking change>.<feature added>.<fix/minor change>
-version = "1.1.4"
+version = "1.1.5"
```

Comment:
do we need to update this?

Reply:
I check it one more time and it's fine with ```1.1.4```. I revert it and we check on CI.